docs/iot-devkit/devkit-get-started.md

```diff
@@ -32,27 +32,46 @@ To connect the IoT DevKit to your computer:
 
 ![Hardware connections](media/iot-devkit-get-started/connect.jpg)
 
-## Configure Wi-Fi
+## Get Started with Azure IoT Hub
 
-IoT projects rely on internet connectivity. Use AP Mode on the DevKit to configure and connect to Wi-Fi.
+1. Follow this [tutorial](https://docs.microsoft.com/en-us/azure/iot-hub/quickstart-send-telemetry-node#create-an-iot-hub) to
+	- Create a new Azure IoT Hub from portal.
+	- Register a new device into Azure IoT Hub and make a note of the device connection string, which looks like:
+	`HostName={YourIoTHubName}.azure-devices.net;DeviceId=MyNodeDevice;SharedAccessKey={YourSharedAccessKey}`
 
-1. Hold down button **B**, push and release the **Reset** button, and then release button **B**. Your IoT DevKit enters AP mode for configuring the Wi-Fi connection. The screen displays the service set identifier (SSID) of the DevKit and the configuration portal IP address:
+	>Please follow the section of **Create an IoT hub** and **Register a device** only. Please skip the section to run the simulated application.
+
+1. Download the latest version of [GetStarted firmware](https://aka.ms/devkit/prod/getstarted/latest) for IoT DevKit which includes the bootloader.
+
+1. Drag and drop the `.bin` file you downloaded from step 1 into the disk named **AZ3166** and wait for IoT Devkit to restart.
```

Comment:
one step missing:
Connect IoT DevKit to your computer via USB
Once the IoT DevKit is connected, in Windows you see a new USB mass storage device in Windows Explorer called AZ3166.